app/src/main/java/com/example/capstone/LoginFragment.java

```diff
@@ -165,4 +167,32 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                 });
         // [END sign_in_with_email]
```

Comment:
This comment doesn't really provide any new info.

I'd suggest revisiting all comments in this file, and keeping only the ones that are absolutely necessary.